src/main/java/fi/hsl/common/mqtt/proto/Mqtt.java

```diff
@@ -147,7 +147,7 @@ private RawMessage(
      * <code>required int32 SchemaVersion = 1 [default = 1];</code>
```

Comment:
Why this file has changed?

Reply:
No idea, maybe generate protos uses some old version of something. Reverting that file.